starter-core/src/main/java/io/micronaut/starter/feature/build/gradle/templates/buildGradle.rocker.raw

```diff
@@ -437,6 +437,27 @@ dependencies {
 @if (features.contains("jmx")) {
     @dependency.template("io.micronaut.jmx", "micronaut-jmx", "implementation", null)
 }
+@if (features.contains("dekorate-kubernetes")) {
+    @dependency.template("io.dekorate", "kubernetes-annotations", "annotationProcessor", "1.0.0")
+    @dependency.template("io.dekorate", "kubernetes-annotations", "implementation", "1.0.0")
+}
+@if (features.contains("dekorate-openshift")) {
+    @dependency.template("io.dekorate", "openshift-annotations", "annotationProcessor", "1.0.0")
+    @dependency.template("io.dekorate", "openshift-annotations", "implementation", "1.0.0")
+
+}
+@if (features.contains("dekorate-knative")) {
+    @dependency.template("io.dekorate", "knative-annotations", "annotationProcessor", "0.12.0")
+    @dependency.template("io.dekorate", "knative-annotations", "implementation", "0.12.0")
+}
+@if (features.contains("dekorate-service-catalog")) {
+    @dependency.template("io.dekorate", "servicecatalog-annotations", "annotationProcessor", "0.12.0")
+    @dependency.template("io.dekorate", "servicecatalog-annotations", "implementation", "0.12.0")
+}
+@if (features.contains("dekorate-halkyon")) {
+    @dependency.template("io.dekorate", "halkyon-annotations", "annotationProcessor", "0.12.0")
+    @dependency.template("io.dekorate", "halkyon-annotations", "implementation", "0.12.0")
+}
```

Comment:
Please move Gradle annotationProcessor dependencies to this file: https://github.com/micronaut-projects/micronaut-starter/blob/1a8578c2400dfbe3f5fb900ddbe434b1c168d9f1/starter-core/src/main/java/io/micronaut/starter/feature/build/gradle/templates/annotationProcessors.rocker.raw

Reply:
done